calico-enterprise/reference/installation/_api.mdx

```diff
@@ -4478,7 +4478,7 @@ _Appears in:_
 
 | Field | Description |
 | --- | --- |
-| `name` _string_ | Name is an enum which identifies the Manager Deployment container by name.<br />Supported values are: tigera-voltron, tigera-manager, tigera-ui-apis, and tigera-es-proxy (deprecated). |
+| `name` _string_ | Name is an enum which identifies the Manager Deployment container by name.<br />Supported values are: calico-voltron, calico-manager, calico-ui-apis |
```

Comment:
This page is generated from the codebase. You'll need to find these texts there and modify accordingly. @Josh-L